plugins/woocommerce/client/legacy/css/twenty-twenty-one.scss

```diff
@@ -8,7 +8,7 @@ $headings: var(--heading--font-family);
 $body: var(--global--font-secondary);
 
 $body-color: currentColor;
-$highlights-color: #88a171;
+$highlights-color: var(--wc-highlight, #777335);
```

Comment:
Adjustment for Twenty Twenty-One theme.

**Before**
![Screenshot 2024-10-16 at 17 55 49](https://github.com/user-attachments/assets/3dd31683-2ce7-4a18-86e7-e45d4500f4b4)

**After**
![Screenshot 2024-10-16 at 17 56 50](https://github.com/user-attachments/assets/d65861a6-e0c2-43a2-b24f-3ac1ab03cc60)